extensions/resteasy-reactive/quarkus-jaxrs-client/deployment/src/main/java/io/quarkus/resteasy/reactive/client/deployment/JaxrsClientProcessor.java

```diff
@@ -111,6 +115,15 @@ void addFeature(BuildProducer<FeatureBuildItem> features) {
         features.produce(new FeatureBuildItem(RESTEASY_REACTIVE_JAXRS_CLIENT));
     }
 
+    @BuildStep
+    void registerClientResponseBuilder(Capabilities capabilities,
+            BuildProducer<ServiceProviderBuildItem> serviceProviders) {
+        if (!capabilities.isPresent(Capability.RESTEASY_REACTIVE)) {
+            serviceProviders.produce(new ServiceProviderBuildItem(ResponseBuilderFactory.class.getName(),
```

Comment:
Are you sure this does what you think it does?

Reply:
care to elaborate?

Reply:
Looking at how `ServiceProviderBuildItem` is handled, it doesn't look like it actually adds the class to a service file. So unless I am reading it wrong, the `ClientResponseBuilderFactory` will never get registered with the service loader.

Reply:
true, `Note that the service interface descriptor file has to exist and match the list of specified provider class names.`

I had a test dependency on reactive-resteasy-jackson in my test and that's why it worked for me. Fixing